### PR TITLE
Add barebones Compose screen

### PR DESCRIPTION
**Pull request by vanwagonet**
refs: MBL-13695
affects: Parent
release note: none

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

File: Parent/Parent/Conversations/ComposeViewController.swift [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import UIKit
+import Core
+
+class ComposeViewController: UIViewController, ErrorViewController {
+    weak var bodyMinHeight: NSLayoutConstraint!
```
Comment (bluwave) [inline Parent/Parent/Conversations/ComposeViewController.swift]: Why is this weak?

File: Parent/Parent/Conversations/ConversationListViewController.swift [hunk 2]
```diff
@@ -21,6 +21,7 @@ import Core
 
 class ConversationListViewController: UIViewController {
     @IBOutlet weak var activityIndicatorView: UIActivityIndicatorView!
+    @IBOutlet weak var composeButton: UIButton!
```
Comment (bluwave) [inline Parent/Parent/Conversations/ConversationListViewController.swift]: I added a button class for this to reuse https://github.com/instructure/canvas-ios/pull/911/files#diff-b92eb2ec71e120917ae5b36055c17dc5

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1015663&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|23
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Parent

[MBL-13695](https://instructure.atlassian.net/browse/MBL-13695)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.32% | 93.33% | -0.01%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Parent/Parent/Conversations/ComposeViewController.swift (hunk 1 above)
Comment (vanwagonet) [reply Parent/Parent/Conversations/ComposeViewController.swift]: It's not in IB, and the constraints are held strongly by views. I don't think there would be a loop if it was a strong reference, though so I can change it if you think that's better.

File: Parent/Parent/Conversations/ConversationListViewController.swift (hunk 2 above)
Comment (vanwagonet) [reply Parent/Parent/Conversations/ConversationListViewController.swift]: The shadow & icon are different, plus it's named "Reply". Also it didn't exist yet. ;) Would you like me to adapt it to a more generic `FloatingButton`?

Comment (bluwave) [reply Parent/Parent/Conversations/ConversationListViewController.swift]: Ah, I didn't realize the icon was different.  If my shadow is off, I think they should be same.  Also, I'd make sure it keeps the shadow path and rasterizing so it doesn't make the scrolling choppy.

File: Parent/Parent/Conversations/ComposeViewController.swift (hunk 1 above)
Comment (bluwave) [reply Parent/Parent/Conversations/ComposeViewController.swift]: Just curious about it.

**Comment by bluwave** [comment]
Nice

**Pull request merged**